cogs/seasonstats.py

```diff
@@ -521,6 +519,129 @@ def __init__(self, bot):
         p = SeasonStatsPaginator(ctx, entries=entries)
         await p.paginate()
 
+    @season_stats.command(name='attacks')
+    async def season_stats_attacks(self, ctx, season: typing.Optional[int] = None):
+        """Get attack wins for all clans.
+
+           By default, you shouldn't need to call these sub-commands as the bot will
+           parse your argument and direct it to the correct sub-command automatically.
+
+           **Example**
+           :white_check_mark: `+season stats attacks`
+        """
+        season = season or await self.bot.seasonconfig.get_season_id() - 1
+        query = """SELECT player_tag, end_attacks - start_attacks as attacks, trophies 
+                        FROM players 
+                        WHERE season_id = $1 AND guild_id = $2
+                        ORDER BY attacks DESC
+                        LIMIT 15
+                    """
+        fetch = await ctx.db.fetch(query, season, ctx.guild.id)
+        table = CLYTable()
+        table.title = f"Attack wins for Season {season}"
```

Comment:
This is my fault but `table.title` does nothing for these renders, you're gonna have to set it in the embed (title="blah")

Reply:
fixed